modules/msgpack/documentation/example.ark

```diff
@@ -0,0 +1,14 @@
+{
+	(import "libmsgpack.so")
```

Comment:
Should be "msgpack.bin" since there is a rewrite rule in the CMakeLists.txt for the module